frontend/src/pages/ActivityReport/Pages/Review/Approver/Approved.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
This is a copy from the `Submitter/Approved` component. I copied instead of creating a common component because I imagine what we want to show about approved reports for the authors vs the approving manager will eventually diverge. I can move them into a single component if any thinks having two separate `Approved` components is premature.

Reply:
It might be a bit premature, but will it introduce a lot of future maintenance overhead? I don't feel like it will, so I think it is ok.

Reply:
I am good with not re-using if it's easier, but that's a good question whether there will be a need to diverge significantly the content for authors vs managers.